src/util/image/DecorationPlacer.java

```diff
@@ -700,23 +700,8 @@ private void fillCurrentImagePointsBasedOnImageFolder(final boolean pointsAreExa
    * @param java
    *        .awt.point p a point on the map
    */
-  private String findTerritoryName(final Point p) {
-    String seaName = null;
-    // try to find a land territory.
-    // sea zones often surround a land territory
-    for (final String name : m_polygons.keySet()) {
-      final Collection<Polygon> polygons = m_polygons.get(name);
-      for (final Polygon poly : polygons) {
-        if (poly.contains(p)) {
-          if (name.endsWith("Sea Zone") || name.startsWith("Sea Zone")) {
-            seaName = name;
-          } else {
-            return name;
-          }
-        } // if
-      } // while
-    } // while
-    return seaName;
+  private Optional<String> findTerritoryName(final Point p) {
+    return Util.findTerritoryName(p, m_polygons);
```

Comment:
Would it be better to just remove this private method all together and just call the util method inline?